<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="UnimoniSeleniumAutomation.UnimoniSeleniumAutomation.HomePageTest" duration-ms="16350" started-at="2019-09-27T12:46:46Z" finished-at="2019-09-27T12:47:02Z">
    <groups>
    </groups>
    <test name="Command line test 8ecd8302-3277-4695-a543-13fab54070b2" duration-ms="16350" started-at="2019-09-27T12:46:46Z" finished-at="2019-09-27T12:47:02Z">
      <class name="UnimoniSeleniumAutomation.UnimoniSeleniumAutomation.HomePageTest">
        <test-method status="PASS" signature="homePageTC()[pri:0, instance:UnimoniSeleniumAutomation.UnimoniSeleniumAutomation.HomePageTest@48974e45]" name="homePageTC" duration-ms="16349" started-at="2019-09-27T18:16:46Z" finished-at="2019-09-27T18:17:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- homePageTC -->
      </class> <!-- UnimoniSeleniumAutomation.UnimoniSeleniumAutomation.HomePageTest -->
    </test> <!-- Command line test 8ecd8302-3277-4695-a543-13fab54070b2 -->
  </suite> <!-- UnimoniSeleniumAutomation.UnimoniSeleniumAutomation.HomePageTest -->
  <suite name="UnimoniSeleniumAutomation.UnimoniSeleniumAutomation.HelloWorldTest" duration-ms="128" started-at="2019-09-27T12:46:45Z" finished-at="2019-09-27T12:46:46Z">
    <groups>
    </groups>
    <test name="Command line test d9aa1316-ca71-4a5b-b0e1-c97bd684f35f" duration-ms="128" started-at="2019-09-27T12:46:45Z" finished-at="2019-09-27T12:46:46Z">
      <class name="UnimoniSeleniumAutomation.UnimoniSeleniumAutomation.HelloWorldTest">
        <test-method status="FAIL" signature="testHelloWorld()[pri:0, instance:UnimoniSeleniumAutomation.UnimoniSeleniumAutomation.HelloWorldTest@120f102b]" name="testHelloWorld" duration-ms="12" started-at="2019-09-27T18:16:46Z" finished-at="2019-09-27T18:16:46Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Hello World] but found [Helo World]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Hello World] but found [Helo World]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at UnimoniSeleniumAutomation.UnimoniSeleniumAutomation.HelloWorldTest.testHelloWorld(HelloWorldTest.java:11)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:91)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:204)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:107)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testHelloWorld -->
      </class> <!-- UnimoniSeleniumAutomation.UnimoniSeleniumAutomation.HelloWorldTest -->
    </test> <!-- Command line test d9aa1316-ca71-4a5b-b0e1-c97bd684f35f -->
  </suite> <!-- UnimoniSeleniumAutomation.UnimoniSeleniumAutomation.HelloWorldTest -->
</testng-results>
